tests/scripts/pre-validation-deploy-slz-roks-and-obs-instances.sh

```diff
@@ -47,15 +44,11 @@ TF_VARS_FILE="terraform.tfvars"
         --arg cluster_id_value "${cluster_id_value}" \
         --arg cluster_resource_group_id_var_name "${cluster_resource_group_id_var_name}" \
         --arg cluster_resource_group_id_value "${cluster_resource_group_id_value}" \
-        --arg log_analysis_ingestion_key_var_name "${log_analysis_ingestion_key_var_name}" \
-        --arg log_analysis_ingestion_key_value "${log_analysis_ingestion_key_value}" \
         --arg cloud_monitoring_access_key_var_name "${cloud_monitoring_access_key_var_name}" \
         --arg cloud_monitoring_access_key_value "${cloud_monitoring_access_key_value}" \
-        --arg log_analysis_instance_region_var_name "${log_analysis_instance_region_var_name}" \
-        --arg log_analysis_instance_region_var_value "${REGION}" \
         --arg cloud_monitoring_instance_region_var_name "${cloud_monitoring_instance_region_var_name}" \
         --arg cloud_monitoring_instance_region_var_value "${REGION}" \
-        '. + {($region_var_name): $region_var_value, ($cluster_id_var_name): $cluster_id_value, ($cluster_resource_group_id_var_name): $cluster_resource_group_id_value, ($log_analysis_instance_region_var_name): $log_analysis_instance_region_var_value, ($log_analysis_ingestion_key_var_name): $log_analysis_ingestion_key_value, ($cloud_monitoring_instance_region_var_name): $cloud_monitoring_instance_region_var_value, ($cloud_monitoring_access_key_var_name): $cloud_monitoring_access_key_value}' "${JSON_FILE}" > tmpfile && mv tmpfile "${JSON_FILE}" || exit 1
+        '. + {($region_var_name): $region_var_value, ($cluster_id_var_name): $cluster_id_value, ($cluster_resource_group_id_var_name): $cluster_resource_group_id_value, ($cloud_monitoring_instance_region_var_name): $cloud_monitoring_instance_region_var_value, ($cloud_monitoring_access_key_var_name): $cloud_monitoring_access_key_value}' "${JSON_FILE}" > tmpfile && mv tmpfile "${JSON_FILE}" || exit 1
```

Comment:
you will also now need to add the trusted profile Id and the ingress endpoint, as they will be required inputs for validation